doc/manuals/admin/database_model.md

```diff
@@ -322,6 +322,8 @@ notifications. It is a number between 0 and 1800000. If defined to 0 or omitted,
 -   **payload**: optional field to store the payload for notification customization functionality in NGSIv2. If
     its value is `null` means that no payload has to be included in the notification. If its value is `""` or if
     the field is omitted, then the NGSIv2 normalized format is used.
+-   **json**: optional field to store a JSON object or array to generated JSON-based payload for
+    notification customization functionality in NGSIv2. More detail of this functionality [here](../users/ngsiv2_implementation_notes#)
```

Comment:
```suggestion
    notification customization functionality in NGSIv2. More detail of this functionality [here](../user/ngsiv2_implementation_notes#)
```

Reply:
Fixed in [8680cd3](https://github.com/telefonicaid/fiware-orion/pull/4157/commits/8680cd37343ff830d99540c36933a14f5d2fd654)